proto/sentry_protos/sentry/v2/taskbroker.proto

```diff
@@ -0,0 +1,124 @@
+syntax = "proto3";
+
+package sentry_protos.sentry.v2;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+enum OnAttemptsExceeded {
+  ON_ATTEMPTS_EXCEEDED_UNSPECIFIED = 0;
+  ON_ATTEMPTS_EXCEEDED_DISCARD = 1;
+  ON_ATTEMPTS_EXCEEDED_DEADLETTER = 2;
+}
+
+message RetryState {
+  // Current attempt number
+  uint32 attempts = 1;
+
+  // The classname or adapter type for the retry policy
+  string kind = 2;
```

Comment:
We haven't found a scenario where we need this yet. Until we have that scenario we could omit this attribute. Adding new attributes in the future is simple enough.